test/server/routes/auth.test.js

```diff
@@ -406,6 +407,27 @@ describe('AUTH', () => {
           done();
         });
     });
-
+  });
+  describe('Pagination router', () => {
+    it('It should get book with query string page and limit', (done) => {
+      chai.request(server)
+        .get(`${BOOKS_BASE_URL}?page=1&limit=10`)
```

Comment:
I saw you added a validator which is great but what I was talking about is a validation that checks the query parameters and test for them. For example, what happens if I tried to access the url by writing the following
* `/api/v1/books?page=today&limit=tomorrow` or
* just the limit parameter like `api/v1/books?limit=10` and similar queries,

Reply:
![image](https://user-images.githubusercontent.com/45515619/59558042-bc883480-8fe0-11e9-856c-f05635ff9d29.png)

Reply:
please, can we try to run the server locally before leaving a comment? because if you did this comment would have been prevented.

Reply:
Alright